docs/website/docs/dlt-ecosystem/destinations/snowflake.md

```diff
@@ -39,11 +39,16 @@ username = "loader"
 host = "kgiotue-wn98412"
 warehouse = "COMPUTE_WH"
 role = "DLT_LOADER_ROLE"
+application = "dltHub_dlt"
 ```
 In the case of Snowflake, the **host** is your [Account Identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier). You can get it in **Admin**/**Accounts** by copying the account URL: https://kgiotue-wn98412.snowflakecomputing.com and extracting the host name (**kgiotue-wn98412**).
 
 The **warehouse** and **role** are optional if you assign defaults to your user. In the example below, we do not do that, so we set them explicitly.
 
+:::note
+We use `application = "dltHub_dlt"` to let Snowflake to know about dlt users if you would like to disable this behavior
+please set it to empty string `application = ""` this will disable it.
+:::
```

Comment:
@VioletM @rudolfix is it fine to word instructions to disable it this way?

Reply:
@sultaniman @VioletM I agree. And btw. this is not how application id is even used :) it is used to get help from Snowflake SE on customer request. IMO we should just describe what it does: that this field sends application id to snowflake, the default value is ... and that you can disable it by setting "" and point to Snowflake documentation.

@VioletM you are free to add commit to this PR!

Reply:
I see I thought it is mostly used for tracking.